### DISCUSSION TITLE
Question on Svelte Society talk from July 4

### DISCUSSION THREAD

**Discussion by nikfp** [jycouet/kitql#168] [Q&A]
Hi @jycouet and thanks for speaking at Svelte Society Austria. I watched your talk on YT and I'm excited about what you unveiled. 

I noticed at the end of your talk that you had a module set up with all the parts needed in KitQL, and it appeared that you had a partial Prisma file for the User module. Any chance you could post a link to the example repo you were using? I'd love to see how you set that up.

**Comment by jycouet** [comment]
Very good notice 👍

The closest example today is: [demo 2](https://www.kitql.dev/docs/demos/03_demo-02) which you can find in the doc (and you can launch it on Stackblitz with one click! 🎉)

For the `FULL` example with partial Prisma and everything, you will have to wait a bit more. 
_FYI, I'm using it in production, it's all about time 😅_
